### ISSUE TITLE
BERT text classification example

### ISSUE THREAD

**Issue by adendek** [marcotcr/lime#448]
I've been working on a text classification problem and my solution utilizes the Bert model implemented in PyTorch. I have a complete example of how to build, train and explain it via lime.  I am wondering whether it is a need for such an example.

**Comment by ccarmel** [comment]
Hi!
I would be happy to see your implementation, 
specifically, I am wondering what did you use for masking inside lime ('UNKWORDZ'? '[MASK]'? '[PAD]'? attn_mask tensor solution? something else I haven't thought about?)
and if you had any problems with bert "guessing" the lime-masked words (as supposedly bert was pre-trained to do so) causing problems with LIME's importance prediction.

Thanks!

**Comment by mbednarski** [comment]
Yes, there definitely is a need! Is your code open source? I'm currently working on this topic and I would love to see your work

**Comment by adendek** [comment]
Right now I have a very basic solution that explains a word level, not token which will be more interesting. 
I'll be working on explanations on a tokens level in about 1 month when I finish my thesis. After that, I'll publish my solution.

**Comment by shimsan** [comment]
> Right now I have a very basic solution that explains a word level, not token which will be more interesting.
> I'll be working on explanations on a tokens level in about 1 month when I finish my thesis. After that, I'll publish my solution.

Very interested in this, if you could post even your word-level code. I'm trying to get something going for non-Transformer, and maybe this could help. Thanks!

**Comment by Tingsie** [comment]
> Right now I have a very basic solution that explains a word level, not token which will be more interesting.
> I'll be working on explanations on a tokens level in about 1 month when I finish my thesis. After that, I'll publish my solution.

Hello, Can you use lime to explain the results of Bert (using tensorflow) text classification? I can't find it in your homepage. Do you open source code? I don't know how to solve this problem.

**Comment by beyondguo** [comment]
So? Where?
